### DISCUSSION TITLE
Oneof  result

### DISCUSSION THREAD

**Discussion by sharonlee12** [Project-MONAI/MONAI#7762] [General]
hello，I want to know what the monai Oneof finally choose?
`transforms.OneOf(transforms=[                            
            monai.transforms.RandCropByPosNegLabeld(keys=["image", "label"], label_key="label",
                                                    spatial_size=(args.roi_x, args.roi_y, args.roi_z),
                                                    pos=2, neg=1,
                                                    num_samples=args.num_samples, image_key="image",
                                                    image_threshold=0),
            monai.transforms.Resized(keys=["image", "label"], spatial_size=(args.roi_x, args.roi_y, args.roi_z))
],
                weights=[1, 1]
            ),`
this is my code,how can i know what it chooses?Crop or resize?
Hope someone will help me!

**Comment by Pkaps25** [comment]
`OneOf` is a [`TraceableTransform`](https://docs.monai.io/en/stable/transforms.html#monai.transforms.TraceableTransform). The object returned by the transformation has an `applied_operations` property.

**Comment by sharonlee12** [comment]
> `OneOf` is a [`TraceableTransform`](https://docs.monai.io/en/stable/transforms.html#monai.transforms.TraceableTransform). The object returned by the transformation has an `applied_operations` property.

Thank you，besides may I ask if you know the position of RandCropByPosNegLabel cropping can be returned? Or is there a cropping transform that makes it easier to obtain the cropped position?
